srml/support/src/traits.rs

```diff
@@ -1,78 +0,0 @@
-//pub trait LockRate {
-//    //TODO： ugly to use u64, ready for hacking
-//    //    type Balance: SimpleArithmetic + As<usize> + As<u64> + Codec + Copy + MaybeSerializeDebug + Default;
-//
-//    fn bill_lock_rate() -> Perbill;
-//
-//    fn update_total_lock(amount: u64, is_add: bool) -> Result;
-//}
-//
-//pub trait DarwiniaDilution<Balance> {
-//    fn on_dilution(treasury_income: Balance);
-//}
-
-use srml_support::traits::Currency;
-
-pub trait OnMinted<Balance> {
-	fn on_minted(value: Balance);
-}
-
-pub trait OnAccountBalanceChanged<AccountId, Balance> {
-	fn on_changed(who: &AccountId, old: Balance, new: Balance);
-}
-
-/// A more powerful lockable currency.
-pub trait LockableCurrency<AccountId>: Currency<AccountId> {
```

Comment:
In support module, I want to make things more generic so other people can reuse this instead design a new trait like us.

Reply:
Moving the traits to mod trait to follow the substrate conventions.

I put them on the same file because current support module are quite simple, which can be splitted to other file if code go large.